### ISSUE TITLE
SQLite3.dll does not exist

### ISSUE THREAD

**Issue by dexusnl** [aviaryan/Clipjump#117]
Hi Avi,

If I clone the latest git code and try to run it I get the following error:
DLL C:\pathtoclipjump\SQLite3.dll does not exist!

It's a fresh copy and the dll file is downloaded to the folder as well.

Best regards,
Rob

**Comment by aviaryan** [comment]
I can't reproduce the issue.
1. Can you post a screenshot of the error you get.
2. Are you copying old Clipjump config (cache dir, settings.ini) to the directory before starting the *latest* Clipjump.
3. Try downloading the zip from https://github.com/aviaryan/Clipjump instead of cloning.

**Comment by dexusnl** [comment]
1.  http://i.imgur.com/TYgZlep.png
2.  no, clean install
3.  that is the one I am using

**Comment by aviaryan** [comment]
4. Check for dll file blocked by Windows. Right click on dll -> properties -> general -> unblock. ([stack overflow](http://superuser.com/questions/590787/what-does-it-mean-when-a-file-is-blocked-in-windows))
5. Although windows is case-insensitive, try renaming `sqlite3.dll` to `something.dll` and then to `SQLite3.dll`

**Comment by dexusnl** [comment]
1. File wasn't blocked
2. Tried to rename it, but didn't make a difference

I did manage to find a different copy of sqlite.dll in my Blender program folder **which does work**.
Replaced that file with the one in the clipjump folder.

Also tried to download the latest version from the sqlite.org website, which gave the following error:
`Version 3.12.0 of sqlite3.dll is not supported!
You can download the current version from www.sqlite.org!`

**Comment by droidgren** [comment]
I have the problem as well. Running Windows 7 x64 (Enterpise). I have also tried the steps u mentioned, and they did not work.

**Comment by aviaryan** [comment]
> I did manage to find a different copy of sqlite.dll in my Blender program folder which does work. :-)
Replaced that file with the one in the clipjump folder.

Try this sqlite3.dll http://rghost.net/6zxVBycNv
It is the same version as the one you cloned/downloaded but is not UPX'ed. 

@droidgren 
Make sure you are running the downloaded Clipjump.ahk using the v12.3 Clipjump.exe (which is 32 bit). The sqlite3.dll is 32 bit so it may not work if AutoHotkey.exe is 64 bit.

**Comment by droidgren** [comment]
@aviaryan The standard version works fine.. I was trying the non-comnpiled git-clone version. But you are right. I had Autohotkey 64 bit installed. When I installed the 32-bit version it worked fine without the "SQLite3.dll does not exist!" error. So that's it !

**Comment by dexusnl** [comment]
@aviaryan That one doesn't work either.
Looks like we found the problem though, I am using 64bit autohotkey just like droidgren.
The blender sqlite3.dll must have worked because it is 64bit as well.

**Comment by aviaryan** [comment]
Thank you guys for helping me find the bug. Cheers. 

PS - It seems Clipjump is working well with AHKx64. Last time I tried, some gui components won't load correctly. I should ship a 64-bit version too then.

**Comment by rubisco57** [comment]
Hi Avi Aryan, 
I stumbled upon your project - I might become of great use to me, thank you for providing this project as open source!
Launching the x64 version I had the same SQLlite3 error as reported above. I resolved it by replacing the Class:SQLiteDB (https://raw.githubusercontent.com/AHK-just-me/Class_SQLiteDB/master/Sources_v1.1/Class_SQLiteDB.ahk) file and downloading the latest SQLlite3.dll -- you might want to update these in a upcoming release. 
Best regards!
